requirements.txt

```diff
@@ -1,8 +1,9 @@
+netaddr==0.7.19
 argparse==1.2.1
 funcsigs==1.0.2
 paramiko==1.16.0
 pyparsing==2.0.3
-python2-pythondialog==3.0.1
+pythondialog==3.0.1
```

Comment:
Hm, the ``pythondialog`` package breaks under Python 2.7. I see you used requirements specifiers in ``setup.py`` to install ``pythondialog`` on Python 3 and ``python2-pythondialog`` on Python 2. Can we do the same in ``requirements.txt``?

Reply:
I'll check that.

Reply:
fixed in 8f45145